src/js/menu/MenuBar.js

```diff
@@ -1,32 +1,29 @@
 L.DNC = L.DNC || {};
 L.DNC.MenuBar = L.Class.extend({
 
-    // defaults
-    options: {
-        parentId : 'menu-bar'
+    initialize: function ( options ) {
+        L.setOptions(this, options);
+        this.children = [];
+        this.domElement = this._buildDomElement();
     },
 
-    initialize: function ( layerlist, options ) {
-        // override defaults with passed options
-        this.layerlist = layerlist;
-        L.setOptions(this, options);
+    // Add object as child. Object must have domElement property.
+    addChild: function ( child ) {
+        this.domElement.appendChild(child.domElement);
+        child.parentDomElement = this;
+        this.children.push( child );
```

Comment:
Every menu object should now have an array of child attributes at `this.children`.  Each of these children are objects, all of which should have a `domElement` attribute.